app/models/miq_server.rb

```diff
@@ -115,6 +118,12 @@ def validate_is_deleteable
     end
   end
 
+  def worker_management
```

Comment:
think using a noun here reads better `worker_manager` (even if the class name stays the same`

Reply:
Yeah I think eventually we'll want to extract this to its own plugin, `manageiq-orchestrator` or something?

Reply:
Okay I renamed this `worker_manager`